strings/strings.ko-KR.json

```
@@ -9170,25 +9170,25 @@
-  "wiki_resets_terraform_para5": "Your planet is not persistent and will be lost if you complete any reset that changes planets.",
-  "wiki_resets_terraform_para6": "Reaching Terraforming can be brutally hard, you should not try it unless you have a lot of prestige.",
-  "wiki_resets_terraform_para7": "Achieving silver star %0 will let the last planet you designed in each universe be found randomly in the future.",
-  "wiki_resets_terraform_para8": "You can add %0 unique planet to the pool per universe.",
-  "wiki_resets_terraform_para9": "Truepath Orbit Decay gives you more planetary points to spend but is very hard, you should not try it.",
-  "wiki_resets_terraform_para10": "Your planet will have a one time bonus called %0.",
-  "wiki_resets_terraform_para11": "The rejuvenated effect is only valid for the next run while the planet is still fresh.",
-  "wiki_resets_terraform_para12": "The rejuvenated effect has the following properties:",
-  "wiki_resets_terraform_para13": "Boosts some labor jobs by %0 (Cement Workers, Miners, Lumberjacks).",
-  "wiki_resets_terraform_para14": "Boosts Army rating by %0.",
-  "wiki_resets_terraform_para15": "Boosts soldier healing (strength depends on highest level of Orbit Decay challenge completed).",
-  "wiki_resets_terraform_para16": "Boosts global production by %0-%1 (decays over time).",
-  "wiki_resets_terraform_para17": "Reduces unsuited penalties by %0.",
-  "wiki_resets_terraform_para18": "Increases extremity of various planetary effects.",
-  "wiki_resets_terraform_para19": "%0 counts as a Greatness event.",
+  "wiki_resets_terraform_para1": "%0은 %1 도전을 플레이하는 중에만 나타나는 특별한 재시작 선택지입니다.",
+  "wiki_resets_terraform_para2": "이 재시작은 %0을 대체하며 동등한 등급의 재시작으로 간주됩니다.",
+  "wiki_resets_terraform_para3": "테라포밍 재시작으로는 %0, %1, %2가 주어집니다.",
+  "wiki_resets_terraform_para4": "커스텀 행성을 설계할 기회도 주어질 것입니다.",
+  "wiki_resets_terraform_para5": "행성은 영구적이지 않고 행성을 변경하는 재시작을 하면 소실됩니다.",
+  "wiki_resets_terraform_para6": "테라포밍에 도달하기는 잔인하리만치 어려울 수 있으며, 충분한 환생 재화를 획득하기 전까지는 시도하지 말아야 합니다.",
+  "wiki_resets_terraform_para7": "은별 %0를 달성하면 앞으로는 각 우주에서 마지막으로 설계한 행성을 발견할 수도 있습니다.",
+  "wiki_resets_terraform_para8": "우주마다 행성 풀에 고유 행성을 %0개씩 넣을 수 있습니다.",
+  "wiki_resets_terraform_para9": "참된 길 궤도 감쇠로는 더 많은 행성 점수를 획득하여 사용할 수 있지만 매우 어려우므로, 도전하지 않는 편이 낫습니다.",
+  "wiki_resets_terraform_para10": "새 행성은 %0이라는 일회성 보너스를 받게 됩니다.",
+  "wiki_resets_terraform_para11": "이 보너스는 행성이 다음 플레이에서 행성이 그대로 살아있는 상태일 동안만 적용됩니다.",
+  "wiki_resets_terraform_para12": "보너스는 다음과 같은 효과를 부여합니다.",
+  "wiki_resets_terraform_para13": "몇몇 노동 직업 보너스 %0 (시멘트 생산공, 광부, 벌목꾼).",
+  "wiki_resets_terraform_para14": "군대 등급 %0 증가.",
+  "wiki_resets_terraform_para15": "병사 회복 증가 (효과는 완료한 궤도 감쇠 도전 가운데 최대 등급에 따름).",
+  "wiki_resets_terraform_para16": "전역 생산 증가 %0-%1 (시간에 따라 감소).",
+  "wiki_resets_terraform_para17": "부적합 불이익 %0 감소.",
+  "wiki_resets_terraform_para18": "다양한 행성 효과의 극단을 증가.",
```

"다양한 행성 효과의 극단을 증가" 에서 극단이 증가한다는 말이 어떤 것을 나타내는 것인지 잘 모르겠습니다. 극단성 혹은 극단율 같은 단어로 이어주는 것이 좋을까 하는데요, 혹시 자원 페널티나 보너스, 초원 행성인 경우 농업 증가율 같은 것이 극단적으로 증가한다는 뜻인가요? 아니면 극단성이 증가한다는 뜻인가요...?

---

(초원 등의) 행성효과에 적용되는 수치가 증가한다는걸로 이해했습니다. 코드를 확인해보니 많이 증가하는 건 아니고 전반적으로 조금씩 상향되네요.

```suggestion
  "wiki_resets_terraform_para18": "다양한 행성 효과의 수치가 양수 방향으로 증가.",
```

---

제가 테라포밍 재시작까지는 가지 못해서 확인이 어렵습니다만 extremity라는 다른 단어를 찾아보니까 무언가에 극단적으로 특화된 것을 나타내는 것 같네요. 다각형 그래프를 찾아보실때 한쪽으로 뾰족하게 나 있는 그런 걸 말하는 것 같기도 합니다.

다만 코드가 많이 증가하는 건 아니고 전반적으로 조금씩 상향된다는 것을 생각하면 조금은 아닌 것 같은데요, 수치가 양수 방향으로 증가한다는 뜻은 "행성의 긍정적인 특화 수치가 증가" 라는 것을 뜻하는 것 같습니다.

---

부정적인 수치는 절대값이 감소하고 긍정적인 수치는 절대값이 증가하는 방향으로 움직이더라고요